riotfile.py

```diff
@@ -178,7 +178,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             venvs=[
                 Venv(
                     name="codespell",
-                    command='codespell --skip="ddwaf.h" ddtrace/ tests/',
+                    command='codespell --skip="ddwaf.h" ddtrace/ tests/ -I docs/spelling_wordlist.txt',
```

Comment:
can you introduce this change in it's own PR?

Reply:
Yup. Removed.